fetch/fetch.go

```diff
@@ -218,15 +222,23 @@ func (f *Fetcher) FetchChainData(ctx context.Context) error {
 				}
 
 				f.logger.Info(
-					"Saved block and tx data for range",
+					"Added to batch block and tx data for range",
```

Comment:
What do you think about keeping this as "Saved", since it's an INFO log?

Reply:
Now we are not saving until we commit the batch. I changed this message to avoid misunderstandings.